### PR TITLE
Fix: Deep Pagination does not append extra parameters anymore

### PR DESCRIPTION
**Pull request by franmomu**
Related with https://github.com/facebook/facebook-php-sdk-v4/issues/483

### CODE REVIEW

File: tests/GraphNodes/GraphEdgeTest.php [hunk 1]
```diff
@@ -84,27 +78,12 @@ public function testCanReturnGraphGeneratedPaginationEndpoints()
         $this->assertEquals('/998899/photos?pretty=0&limit=25&before=foo_before_cursor', $prevPage);
     }
 
-    public function testCanGeneratePaginationEndpointsFromACursor()
```
Comment (SammyK) [inline tests/GraphNodes/GraphEdgeTest.php]: Uh... why did you remove cursor pagination? Lol.

**Comment by SammyK** [comment]
Update: still need to dig into this PR a bit deeper. My hope is this week sometime. :)

**Comment by sardarsaab** [comment]
Is there any option to add call to action using php sdk in facebook page video?
